posthog/hogql_queries/test/__snapshots__/test_events_query_runner.ambr

```diff
@@ -117,7 +117,7 @@
 # ---
 # name: TestEventsQueryRunner.test_presorted_events_table
   '''
-  SELECT tuple(events.uuid, events.event, events.properties, toTimeZone(events.timestamp, 'UTC'), events.team_id, events.distinct_id, events.elements_chain, toTimeZone(events.created_at, 'UTC'))
+  SELECT tuple(events.uuid, events.event, events.properties, toTimeZone(events.timestamp, 'UTC'), events.team_id, events.distinct_id, events.elements_chain, toTimeZone(events.created_at, 'UTC')) AS `tuple(uuid, event, properties, toTimeZone(timestamp, 'UTC'), team_id, distinct_id, elements_chain, toTimeZone(created_at, 'UTC'))`
```

Comment:
Not familiar with this at all so just curious why these changes came up?

Reply:
This is unrelated to my change, had to be some upstream stuff.